classes/game.rb

```diff
@@ -0,0 +1,17 @@
+class Game
+  # Attributes
+  attr_accessor :id, :name, :genre, :multiplayer, :last_played_at
+
+  # Methods
+  def initialize(id, name, genre, multiplayer, last_played_at)
```

Comment:
Please provide these attributes in the constructor:
def initialize(multiplayer, last_played_at, publish_date, archived: false)
or def initialize(multiplayer, last_played_at, publish_date)